src/Microsoft.AspNetCore.Authentication.Abstractions/AuthenticationOptions.cs

```diff
@@ -48,6 +48,19 @@ public void AddScheme(string name, Action<AuthenticationSchemeBuilder> configure
         }
 
         /// <summary>
+        /// Adds an <see cref="AuthenticationScheme"/>.
+        /// </summary>
+        /// <typeparam name="THandler">The <see cref="IAuthenticationHandler"/> responsible for the scheme.</typeparam>
+        /// <param name="name">The name of the scheme being added.</param>
+        /// <param name="displayName">The display name for the scheme.</param>
+        public void AddScheme<THandler>(string name, string displayName) where THandler : IAuthenticationHandler
```

Comment:
But mainly I just wanted the sugar, we had a bug for this that got punted to 2.1, but since I'm here and adding tests :) https://github.com/aspnet/Security/issues/1186